### ISSUE TITLE
Documentation: Error when cloning on Windows

### ISSUE THREAD

**Issue by Guiguerreiro39** [IEEE-VIT/ToDo-iOS#59]
**Describe the bug**
When trying to clone this repo on windows, I've came across an error

``` batch
error: invalid path '.github/pull_request_template.md.'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/
```

**To Reproduce**
1. Be on a Windows machine
2. Fork the repo
3. git clone %repo%

**Expected behavior**
The repository should be cloned

**Solution**
Obviously the issue lies on the *.github/pull_request_template.md.* file so if that file is removed, it should work fine. Otherwise, another possible alternative would be:
1. git init ToDo-iOS && cd ToDo-iOS
2. git config core.protectNTFS false
3. git remote add %repository_url%
4. git fetch
5. git checkout origin/master

**Comment by aaryankotharii** [comment]
Hi @Guiguerreiro39, yes I suppose the error is due to `pull_request_template.md.` and deleting it shoulf fix the issues on windows. Thanks for pointing it out!
Would you like to work on this issue?

**Comment by Guiguerreiro39** [comment]
> Hi @Guiguerreiro39, yes I suppose the error is due to `pull_request_template.md.` and deleting it shoulf fix the issues on windows. Thanks for pointing it out!
> Would you like to work on this issue?

I can work on it.. is that file necessary for anything else?

**Comment by aaryankotharii** [comment]
yes so if you look carefully there are two filed in .github directory, one is `pull_request_template.md.`second is `PULL_REQUEST_TEMPLATE.md`. You can delete the first one, the second one is being used for the template of pull requests.
